src/js/components/jtable/TableCell.jsx

```diff
@@ -24,34 +37,40 @@ export class TableCell extends Component {
             classNames.push(className);
         }
 
+        const {
+            linkProps,
+            tagOrComponent
+        } = generateLink('div', href, linkTo);
```

Comment:
so the whole point of this is to use things other than `div` as the wrapping element, right?